.gitignore

```diff
@@ -0,0 +1,9 @@
+CMakeCache.txt
+CMakeFiles
+CMakeScripts
+Makefile
+cmake_install.cmake
+install_manifest.txt
+*.a
+*.mod
+*.so
```

Comment:
We can probably put `build/` in the .gitignore and remove the CMake files (all the non-starred files above).